apl/scalar-functions/hash-functions/hash.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: hash
+description: 'This page explains how to use the hash function in APL.'
+---
+
+Use the `hash` scalar function to transform any deterministic, hash-able value into a 64-bit signed integer. The result is stable for identical input values across your entire cluster, so you can:
```

Comment:
Probably putting `hash-able` in here makes the definition recursive. I think it probably makes more sense to say like this:

Use the `hash` scalar function to transform any data type as a string of bytes into a signed integer. The result is deterministic so the value is always identical given the same input data.